sources/world/admin_divisions_v1.hjson

```diff
@@ -0,0 +1,171 @@
+{
+    versions: '1'
+    type: http
+    production: true
+    wikidata: Q56061
+    note: 'Administrative divisions of countries collected from Natural Earth and OpenStreetMap'
+    data: ''
+    attribution: [
+      {
+        label: {
+          en: Made with NaturalEarth
+        }
+        url: {
+          en: http://www.naturalearthdata.com/about/terms-of-use
+        }
+      }
+      {
+        label: {
+          en: © OpenStreetMap contributors
+        }
+        url: {
+          en: https://www.openstreetmap.org/copyright
+        }
+      }
+      {
+        label: {
+          en: Elastic Maps Service
+        }
+        url: {
+          en: https://www.elastic.co/elastic-maps-service
+        }
+      }
+    ]
+    fieldMapping: [
+        {
+          type: id
+          name: iso_3166_2
+          desc: ISO-3166-2 identifier
+        }
+        {
+          type: id
+          name: iso2
+          desc: Country code
+        }
+        {
+          type: property
+          name: name
+          desc: Division name
+        }
+    ]
+    name: 'administrative_divisions'
```

Comment:
consider renaming to `Administrative regions`, or `2nd level regions`, ... Something to tie it to ECS more closely.

Reply:
I also prefer something that is close to the wording we see in ECS and the results of the geoip processor. +1 for `Administrative regions`